Automattic/jetpack
# Slideshow block: Remove "fade" transition slide background

#### Changes proposed in this Pull Request:
This PR addresses the issue https://github.com/Automattic/jetpack/issues/12962.

After a discussion on the [issue thread](https://github.com/Automattic/jetpack/issues/12962#issuecomment-916208478), we decided to propose a CSS change that removes the default background color from slides and makes sure that only the active slide is being displayed.

#### Does this pull request change what data or activity we track or use?
No, it does not.

#### Testing instructions:
1. Create a Slideshow Block
2. Change the transition effect to `Fade` and publish the post/page
3. View the post/page. There should be no background around slides, the page/post background color should be used.

Before:
![Markup on 2021-09-10 at 10:06:16](https://user-images.githubusercontent.com/25105483/132821518-9ca8753f-1613-4778-a79c-e5a654e1a4b1.png)

After:
![Markup on 2021-09-10 at 10:05:05](https://user-images.githubusercontent.com/25105483/132821549-0fe50c19-b259-4c5b-8954-20a5fd7246e8.png)

Fixes https://github.com/Automattic/jetpack/issues/12962.
